nextflow_schema.json

```
@@ -378,6 +378,27 @@
                     "description": "Maximum ASV length",
                     "help_text": "Remove ASV that are above the maximum length threshold (default: filter is disabled, otherwise 1000000). Lowering the threshold might reduce false positive ASVs (e.g. PCR off-targets)."
                 },
+                "filter_codons": {
+                    "type": "boolean",
+                    "description": "Filter ASVs based on codon usage",
+                    "help_text": "This setting is to be used when the ASVs need to be filtered if they contain STOP codons in their sequence and they are of length in the multiple of 3.",
+                    "properties": {
+                        "orf_start": {
+                            "type": "integer",
+                            "description": "Starting position to check for the stop codons in the ASVs",
+                            "help_text": "By default, when the `--filter_codons` is triggered, the codons start from the beginning of the ASV sequences. If you would like to change this setting, you can specify from which position of the ASV does the codon triplets be calculated and checked. \n\nPlease note that the length from this position to the end of the ASV or until the `--orf_end` should be a multiple of 3 to make sure the filtering works properly."
+                        },
+                        "orf_end": {
+                            "type": "integer",
+                            "description": "Ending position to check for the stop codons in the ASVs",
```

Why not add here the default with
```suggestion
                            "default": "TAA,TAG",
                            "description": "Ending position to check for the stop codons in the ASVs",
```